builder/vmware/vmx/config.go

```diff
@@ -112,6 +114,15 @@ func (c *Config) Prepare(raws ...interface{}) ([]string, error) {
 		}
 	}
 
+	if c.DiskTypeId == "" {
```

Comment:
If my previous comment is right, this should be moved to `builder/vmware/common/disk_config.go`

Reply:
Because the defaulting depends on RemoteType, which isn't in the disk_config, this needs to live inside of the main configs -- you're right that it was an accident removing it from the iso/config though. I'll fix that.